sql/moz-fx-data-shared-prod/firefox_accounts/nonprod_fxa_content_auth_stdout_events/view.sql

```diff
@@ -105,5 +105,6 @@ SELECT
   JSON_VALUE(event_properties, "$.promotionCode") AS promotion_code,
   JSON_VALUE(event_properties, "$.payment_provider") AS payment_provider,
   JSON_VALUE(event_properties, "$.checkout_type") AS checkout_type,
+  JSON_VALUE(event_properties, "$.source_country") AS source_country,
```

Comment:
Should we delete jsonPayload.fields.country, from LN 12, 30, 48? seems to just be null. Not sure if it has another purpose.

Reply:
it should probably be a separate PR if we do

Reply:
it should probably be a separate PR if we do